spec/requests/api/schemas/json/user_get_response.json

```diff
@@ -459,6 +459,10 @@
             "null"
           ]
         },
+        "sidebar_sections": {
+          "type": "array",
+          "items": {}
+        },
```

Comment:
Is there a way for us to exclude this from the API response? I don't think it is necessary to expose sidebar related attributes in the API at this moment.